audio/Makefile

```diff
@@ -0,0 +1,148 @@
+#
```

Comment:
Can you add a TODO for moving this into a common Makefile too?

A bunch of this probably belongs in the fx2lib-linux-headers repo (IE any of the stuff related to descriptors).